### DISCUSSION TITLE
Is it possible to list open buffers in nvim-tree?

### DISCUSSION THREAD

**Discussion by rasivasu** [nvim-tree/nvim-tree.lua#1102] [Q&A] [answered]
If available, it will be a very valuable feature and save users from typing `:ls<cr>` to view the list of open buffers.

**Comment by kyazdani42** [comment, marked as answer]
hi ! what do you mean by listing ? If it's just the same as the output as `:ls` then i don't think this should be implemented in nvim-tree, although you could bind a mapping to it.

**Reply by rasivasu** [reply]
Yes, I was expecting something like what coc-explorer has.  I have given it here for reference: https://user-images.githubusercontent.com/1709861/76720263-471be100-6777-11ea-82c1-614627097b02.png

**Reply by kyazdani42** [reply]
I see :) something like this could be implemented once the live-filter has been added i think :)

**Comment by alex-courtis** [comment]
A full solution has been provided via a new filter `B`, see `:help filters.no_buffer`
